makepanda/installpanda.py

```diff
@@ -13,8 +13,7 @@
 from optparse import OptionParser
 from makepandacore import *
 
-# DO NOT CHANGE TO sysconfig - see GitHub issue #1230
-from distutils.sysconfig import get_python_lib
+import sysconfig
```

Comment:
There's a reason the comment is here.  This would break when building on older Ubuntu versions.

Reply:
We might as well use `locations.py` here too.

Reply:
I think the difficulty is that we can't import direct modules in makepanda.  It's not immediately obvious to me what the neatest solution is to avoid duplicating that file.